MOTECH-2121: Creates a MOTECH docker image built using the DEB

kmadej opened the pull request:
All needed information about this PR you can find in the 'README' file.

Moreover I corrected misspelling in 'README' file in motech-centos.

Commit:
MOTECH-2121: Creates a MOTECH docker image built using the DEB

All needed information about this PR you can find in the 'README' file.

Moreover I corrected misspelling in 'README' file in motech-centos.

pgesek commented on motech-deb/Dockerfile:
```diff
@@ -0,0 +1,31 @@
+ADD /motech.deb /motech.deb
+RUN dpkg -i motech.deb
+
+EXPOSE 61616 8161
```
any reason for exposing the ActiveMQ port?

pgesek commented on motech-deb/Dockerfile:
```diff
@@ -0,0 +1,31 @@
+# Set some vars so the oracle installer doesn't ask us to accept the license
+RUN echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections
+RUN echo debconf shared/accepted-oracle-license-v1-1 seen true | debconf-set-selections
+RUN apt-get install -y oracle-java8-installer
```
Won't installing motech install java as dependency, just wondering if we need these steps?

kmadej replied:
Motech (from version 0.28) needs Java 8. In deb packaging there is hardcoded path to java-8-oracle. So we need exactly this java. To install java-8-oracle on ubuntu we need all these steps. I checked it one more time to be sure.

Currently there is a problem with deb packaging on the latest master. I will create a ticket for it. However this docker container should work correctly with other motech versions.

pgesek replied:
Looked into this - we should update the JDK list in the daemon script. We cannot use Java 7 after version 0.28.X

Commit:
MOTECH-2121: Removed exposing ActiveMQ port

Commit:
MOTECH-2121: The package installs dependencies

I used ubuntu 15.04 version and now java8 can be easily installed as a package
dependency. The gdebi installs package dependencies (ActiveMQ, tomcat, etc.).

kmadej commented:
Now motech installs java8, activemq, tomcat7 as dependency. This PR is related with https://github.com/motech/motech/pull/512
